src/CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ add_subdirectory(conventional)
 add_subdirectory(ncep)
 add_subdirectory(ssec)
 add_subdirectory(wrfda-ncdiag)
+add_subdirectory(pseudo)
```

Comment:
Is `pseudo` appropriate?  It is not a pseudo observation.
Perhaps `single_observation`.

Reply:
I'm fine with 'single_observation', or maybe even 'singleob' just for brevity's sake

Reply:
I will wait and see what any other reviewers say, as I don't have a strong opinion one way or another.

Reply:
I like the dir name of single_observation because it is quite clear.

Reply:
I fully support and approve the PR. It's a great idea to have this addition.  To have it work with a satellite brightness temperature, does it need potentially another variable to handle channel?

Reply:
I'll change to single_observation and push. As of now, I think this will work for brightness_temperature as we use the convention brightness_temperature_N for N variables. When we switch to brightness_temperature being 2D (nlocs,nchan), then yes, this will probably need to be modified (along with many other converters).